include/triton/core/tritonserver.h

```diff
@@ -1834,6 +1834,15 @@ TRITONSERVER_ServerOptionsSetHostPolicy(
     TRITONSERVER_ServerOptions* options, const char* policy_name,
     const char* setting, const char* value);
 
+/// Set the default maximum batch size when using auto-complete.
```

Comment:
Set the default maximum batch size which may be used by the backends for auto-completion of the model configuration.

Reply:
Updated to new comment